test/spec/coreSpec.js

```diff
@@ -1910,5 +1916,56 @@ describe('undefined - JS to JSON projection / JSON to JS extension', function()
         bar: null
       });
     });
+
+    it(`should allow __proto__ modifications when the flag is set`, function() {
```

Comment:
To me more descriptive
```suggestion
    it(`should allow __proto__ modifications when the mutateDocument flag is set`, function() {
```